added my resume.md file

udayempire opened the pull request:
Here is my resume in markdown format.

Changes I made- 
1. Created a contributiors folder for everyone to add their resume.md file.
2. Added my resume udayempire.md as instructed.

udayempire committed:
udayempire-resume

Eshan-Sharma commented on contributors/udayempire.md:
```diff
@@ -0,0 +1,53 @@
+
+---
+
+## 🚀 PROJECTS
```
Add live links

Eshan-Sharma commented on contributors/udayempire.md:
```diff
@@ -0,0 +1,53 @@
+---
+
+## 💼 WORK EXPERIENCE
+### **Devsthanam, Co-Founder** (Nov 2023 - Present)
```
Liked the current format, can add a description to your experience by adding what you did there, what is the tangible outcome your work did (x% increase in efficiency or pipeline or y% decrease in latency).

udayempire committed:
Update udayempire.md


I have implemented the following improvements:

-Added live project links and GitHub repositories for better accessibility.
-Refined the writing style by incorporating percentages to quantify impact effectively.

udayempire reviewed (commented):
I have implemented the following improvements:

Added live project links and GitHub repositories for better transparency and accessibility.
Refined the writing style by incorporating percentages to quantify impact effectively.

Eshan-Sharma commented:
Good work, approved
